src/auth/react-auth0-spa.tsx

```diff
@@ -73,6 +76,42 @@ export const Auth0Provider = ({
     setIsAuthenticated(true);
     setUser(user);
   };
+
+  useEffect(() => {
```

Comment:
Can we add a comment just above this hook explaining that this is a custom addition to the file that was _not_ copied in from Auth0 docs?